File: Arrays & Strings/newvect.cpp
#include <iostream>
#include <vector>
using namespace std;
int main()
{
    vector<int> vect;
    int number;
    int lim;
    cin >> lim;
    for (int i = 0; i < lim; i++)
    {
        cin >> number;
        vect.push_back(number);
    }
    vect.push_back(23);
    vect.push_back(24);
    vect.push_back(25);
    for (int i = 0; i < vect.size(); i++)
    {
        cout << " " << vect[i];
    }
    return 0;
}

File: Arrays & Strings/newvect.java
import java.util.*;

public class newvect {
    public static void main(String[] args) {
        Vector<Integer> vect = new Vector<>();
        // Declaration method to use Vector in Java
        // In C++ we use vect.pushback(num) and add variable in C++ but in Java we'll
        // use vect.add(num) and add into it's function of add
        for (int i = 0; i < 20; i++) {
            vect.add(i);
        }
        for (int i = 0; i < 20; i++) {
            System.out.print("|" + i);
        }
        System.out.print("|");
        // To get size of this vector till used
        System.out.println("\nSize: " + vect.size());
        // To get capacity of this vector till used
        System.out.println("Capacity: " + vect.capacity());
    }
}
